lib/providers/apps_provider.dart

```diff
@@ -605,6 +628,9 @@ class _APKPickerState extends State<APKPicker> {
       scrollable: true,
       title: const Text('Pick an APK'),
       content: Column(children: [
+        Text(
+          'Your device ${widget.arch == "x86" ? "support" : "supports"}: ${widget.arch ?? 'Unknown'}',
+        ),
         Text('${widget.app.name} has more than one package:'),
```

Comment:
I think we should put the architecture note below this text instead of above it. The architecture note should also have some visual separation - maybe put it in italics or make the font size smaller or both, whatever looks better to you.